modules/related-posts/jetpack-related-posts.php

```diff
@@ -92,23 +143,21 @@ public function __construct( $blog_id_local, $blog_id_wpcom ) {
 	/**
 	 * Add a checkbox field to Settings > Reading for enabling related posts.
 	 *
+	 * @access public
 	 * @action admin_init
 	 * @uses add_settings_field, __, register_setting, add_action
-	 * @return null
+	 * @return void
 	 */
 	public function action_admin_init() {
-
```

Comment:
We intentionally add a line break before an inline comment so please don't remove them.